### ISSUE TITLE
Does numeric.js require additional licenses

### ISSUE THREAD

**Issue by samreid** [phetsims/sherpa#50]
numeric.js says "(See also licenses in the resources subdirectory.)"  Does it require auxiliary licenses?

**Comment by samreid** [comment]
There is only one license file in the resources subdirectory, and it is a copyright for "quadratic programming mainly".  It has a different copyright statement and is also covered by the MIT license, like numeric.js is.  Even though we are not using quadratic programming, we are including that code since I see

```js
function solveQP(Dmat, dvec, Amat, bvec, meq, factorized) {
```
in our numeric.js.  So we should do what we did for #45 and add this additional license statement.

**Comment by samreid** [comment]
Fixed during #45, closing.
